opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/_view_instrument_match.py

```diff
@@ -52,6 +53,38 @@ def __init__(
         self._attributes_previous_point: Dict[frozenset, _PointVarT] = {}
         self._lock = Lock()
         self._instrument_class_aggregation = instrument_class_aggregation
+        self._name = self._view._name or self._instrument.name
+        self._description = (
+            self._view._description or self._instrument.description
+        )
+        if not isinstance(self._view._aggregation, DefaultAggregation):
```

Comment:
Since we have this logic to populate `self._aggregation` now, do we still need to calculate it in `consume_measurement` below?

Reply:
Yes, we need it in both places. We need to create a new instance of `aggregation` [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/_view_instrument_match.py#L79) and [here](https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/_view_instrument_match.py#L85) because each aggregation instance has its own value that is aggregated. `self._aggregation` serves another purpose and is to assist in conflict detection.